### ISSUE TITLE
Just white squares after a day of TRAIN...

### ISSUE THREAD

**Issue by ZeroCool22** [shaoanlu/faceswap-GAN#72]
https://www.youtube.com/watch?v=9MaQYIpxnMY

Training done with: **FaceSwap_GAN_v2_sz128_train.ipynb**

Video: **FaceSwap_GAN_v2_test_video.ipynb**

**Comment by petergerten** [comment]
What did you see while training? Was the preview ok?

**Comment by ZeroCool22** [comment]
Yes, preview windows was ok.

**Comment by petergerten** [comment]
I think you have to do the video generation with the routine inside the 128 notebook, as the others are for the 64px models.

**Comment by ZeroCool22** [comment]
OK, **FaceSwap_GAN_v2_test_video** seems to "works".

https://www.youtube.com/watch?v=tFCDUoPwBJQ

There are still problems with the ALIGN of the faces in some frames.

@petergerten Can you explain me the "routine inside the 128 notebook" i don't see any test_video script with **"128"** in his title on it

**Comment by petergerten** [comment]
@ZeroCool22 the video generation for the 128 notebook is included in the same notebook, there is not separate one. See " Make video clips w/o face alignment" - no alignment available though

**Comment by ZeroCool22** [comment]
@petergerten That's the video generator i used... **w/o face alignment** since the w/aligment said it's experimental.

**Comment by petergerten** [comment]
@ZeroCool22 I do have the alignment problem as well: see https://github.com/shaoanlu/faceswap-GAN/issues/67

**Comment by ZeroCool22** [comment]
@petergerten Yeah it doesn't align well and also sometimes it's ZOOMED in/out, i was really hyped for this REPO, but looks like need some time to be a better/fast and easy alternative to the FakeApp.

**Comment by petergerten** [comment]
@ZeroCool22 same thoughts here :) I still don't quite understand if this zoomed out effect is simply an alignment problem or what exactly is happening

**Comment by shaoanlu** [comment]
In **readme description** it writes sz128 is NOT compatible with test_video notebook. In addition, if you use sz128 model in test_video notebooks, the will be an **error message** in “load model” step. Furthermore, there is a **same issue** you posted before, you can update info there... And if you are not satisfied with the mask, perceptual loss is always there for you to try.

Just, please, provide more detail about your problem like what is the output message of each step or more concretely, what are the previws look like and what are they different from output videos, insread of throwing me a not-working video. And please notice that I can not reproduce your 1.5 days training on 1080ti (it will cost me 3 full days for tantamount training iterrations).

**Comment by shaoanlu** [comment]
To be honest,, this repo is nothing but a personal project. I did not make any cent from it and did not (and will never) ask for donation. I create this repo because I want to verify my understanding on deeplearning and perhaps have chance to interact with those who are also interested in DL (this is why I choose jupyter, which is suitable for fast prototyping). But untill now, I am feeling myself more like doing custom service for a “DL? Who cares?” product while learning very little through out the process. The “deepfakes” community cares much more about “faking” than going “deep”. These deviations really burn me out. I am afraid I can not meet the high expectation of the deepfakes community without others help.

**Comment by ZeroCool22** [comment]
@shaoanlu I would have posted an error message if there was one to show, when i got that just white square there was no error, just the result you see on the video.

If there are errors to show, i ofc. will post these, like i did in my last post about the **color correction issue**.

**Comment by shaoanlu** [comment]
OK, I’ll look into the color correction issue.

**Comment by ZeroCool22** [comment]
@shaoanlu Thx.

**Comment by deepfakesclub** [comment]
@shaoanlu 
You sound like a good guy, or girl but statistically unlikely, so I wanted to speak up in your support. I hope you don't mind.

If you are too nice, people will take advantage of you (not saying this thread starter, he's just excited about the tech, but in general). Don't let yourself burn out because of them. There are many authors of popular githubs that don't reply to messages anymore, probably for similar reasons. The WORST result is if it hurts your love of deep learning.

It's the internet. You spend hours working on something and share it for free. Then, people use it and complain, this sucks, why doesn't it work right, etc. Other people are trying to take your work or other work and profit from it while refusing to acknowledge or thank the original sources.

Frankly, none of these people deserve your time or your talents. Sometimes it is not bad people, but just young, socially insensitive nerds, or culturally different people. Still, no one has a right to ask anything of you.

The challenge of getting something new working is fun. Brainstorming ideas, being awed by the latest great paper, trying out crazy experiments - hell yeah. Polishing a free product for the ungrateful public... not so much.

Maybe just browse and reply here once a week or once a month. Stick with what you really love. If you stop contributing to this repo completely, you have already achieved far more for the deepfaking community than almost anyone else. THEY should be helping YOU, not the other way around!

As you also said, most are more concerned with "faking". Think about the type of people this work attracts. Some programmers I know have stated they are interested in the technology but don't want to be linked with it. That may be part of the reason why you are getting less technical discussions than you had hoped. Also, people became seriously interested in deep learning because of deepfakes, it will still take them many months before they are ready to have a meaningful discussion.

There are definitely cool people around this project, like at the main repo, so I hope you stick around. But maybe keep to your own pace and own interests.

**Comment by shaoanlu** [comment]
@deepfakesclub You’re right. My wishful thinking mislead me not awaring of how sensitive a topic deepfakes is. And I have to change to a more casual mindset for opensource project. Big lessons learned. Thanks.

**Comment by oatssss** [comment]
@deepfakesclub you said:
>Some programmers I know have stated they are interested in the technology but don't want to be linked with it.

Do these people know something I don't? I'm curious to know why they don't want to be linked with this work. To me, it's a very interesting open source project that just happens to be used in a nsfw manner.
